py3status/modules/hueshift.py

```diff
@@ -0,0 +1,191 @@
+# -*- coding: utf-8 -*-
+"""
+Shift color temperature on the screen.
+
+Configuration parameters:
+    button_down: mouse button to change color temperature (default 5)
+    button_reset: mouse button to reset color temperature (default 3)
+    button_up: mouse button to change color temperature (default 4)
+    command: specify blueshift, redshift, or sct to use, otherwise automatic
+        (default None)
+    delta: specify interval value to change color temperature (default 250)
+    format: display format for this module
+        *(default '\?if=disabled&color=bad \u263c!'
+        '|\?color=hue \u263c {temperature}K')*
+    initial: specify initial color temperature to use (default 6500)
+    maximum: specify maximum color temperature to use (default 25000)
+    minimum: specify minimum color temperature to use (default 1000)
+    thresholds: specify color thresholds to use
+        (default [(6499, '#f3c'), (6500, '#ff3'), (6501, '#3cf')])
+
+Control placeholder:
+    {disabled} a boolean based on pgrep processing data
+
+Format placeholder:
+    {temperature} color temperature, eg 6500
+    {command} command name, eg blueshift, redshift, sct
+
+Color options:
+    color_bad: hueshift paused due to running processes
+
+Color thresholds:
+    hue: print a color based on the value of color temperature
+
+Requires:
+    blueshift: an extensible and highly configurable alternative
+    redshift: program to adjust the color temperature of your screen
+    sct: set color temperature
+
+Suggestions:
+    campfire: 4500
+    dust storm on mars: 2000
+    coffee free all nighter: 8000
+
+Examples:
+```
+# good rules; show command name; don't colorize everything
+hueshift {
+    format = '[\?if=disabled&color=bad \u263c!|[\?color=hue&show \u263c '
+    format += '{command}] {temperature}K]'
+    minimum = 3000
+    maximum = 10000
+}
+```
+
+@author lasers
+
+SAMPLE OUTPUT
+# warm
+{'full_text': u'\u263c 3000K', 'color': #f3c},
+
+# neutral
+{'full_text': u'\u263c 6500K', 'color': #ff3}
+
+# cool
+{'full_text': u'\u263c 10000K', 'color': #3cf},
+
+disabled
+{'full_text': u'\u263c!', 'color': #f00}
+"""
+
+STRING_BAD_COMMAND = 'invalid command `%s`'
+STRING_NOT_INSTALLED = 'command `%s` not installed'
+STRING_NOT_AVAILABLE = 'no available command'
+
+
+class Py3status:
+    """
+    """
+    # available configuration parameters
+    button_down = 5
+    button_reset = 3
+    button_up = 4
+    command = None
+    delta = 250
+    format = u'\?if=disabled&color=bad \u263c!|\?color=hue \u263c {temperature}K'
```

Comment:
I still think a nice error rather than having a crazy format would be much simpler for the user.

Reply:
I hate github why is it hiding this comment!

I consider this a blocker

Reply:
I think `self.py3.error()` in this case would be bad because `pgrep` could run into the commands from `hueshift` during scrolling. I confirm I run into this before... I occasionally see `!` there and there.

Reply:
I do not believe this to be an error so I don't want to trigger an error. This is more of a blocker... so users are reminded that they already have something running... which should almost never happen  if they like this module.